lib/Extension/CoreExtension.php

```diff
@@ -344,7 +344,10 @@ private function registerProgressLoggers(Container $container)
         $container->register('progress_logger.registry', function (Container $container) {
             $registry = new LoggerRegistry();
 
-            foreach ($container->getServiceIdsForTag('progress_logger') as $serviceId => $attributes) {
+            /** @var array $serviceIdsForTag */
+            $serviceIdsForTag = $container->getServiceIdsForTag('progress_logger');
```

Comment:
Perhaps this should be amended upstream?

Reply:
hmm, it should already have a docblock return type of `array` at least :thinking: but I wouldn't worry about it for this PR.